lib/takeaway.rb

```diff
@@ -0,0 +1,38 @@
+require 'twilio-ruby'
+
+class Takeaway
+
+  attr_reader :menu
+
+  def initialize
+    @menu = { chicken_noodle_soup: 4.95,
+      duck_spring_rolls: 4.95,
+      crispy_duck: 8.95,
+      singapore_fried_noodles: 7.95,
+      steamed_rice: 2.95 }
+  end
+
+  def complete_order(order_total)
+    send_sms("Thank you! Your order total of £#{order_total} was placed and will be delivered before #{delivery_time}")
+  end
+
+  def send_sms(message_to_send)
+    account_sid = 'ACe54bd6751e630538fe3b6e2e16f46975'
+    auth_token = '04143500350b6edc0ed55c042c3629c9'
+    @client = Twilio::REST::Client.new account_sid, auth_token
+    message = @client.account.messages.create(
+        :from => '441358202027',
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)